recipes/easyloggingpp/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+required_conan_version = ">=1.33.0"
+
+class EasyloggingppConan(ConanFile):
+    name = "easyloggingpp"
+    license = "The MIT License (MIT)"
+    homepage = "https://github.com/amrayn/easyloggingpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Single header C++ logging library."
+    topics = ("logging", "stacktrace", "efficient-logging")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt",
+    options = {
+        "enable_crash_log": [True, False],
+        "enable_thread_safe": [True, False],
+    }
+    default_options = {
+        "enable_crash_log": False,
+        "enable_thread_safe": False,
+    }
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake is not None:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["build_static_lib"] = True
+        self._cmake.definitions["enable_crash_log"] = self.options.enable_crash_log
```

Comment:
Why not set the risk definitions directly?

```suggestion
        self._cmake.definitions["ELPP_FEATURE_CRASH_LOG"] = self.options.enable_crash_log
```

Reply:
cmake definitions/variables != c++ defines

Reply:
🤦‍♂️ Sorry, thanks!